tests/vnc_test.go

```diff
@@ -155,6 +160,49 @@ var _ = Describe("[rfe_id:127][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 			_, err = wrappedRoundTripper.RoundTrip(req)
 			Expect(err).ToNot(HaveOccurred())
 		})
+
+		It("should connect to vnc with --proxy-only flag", func() {
+
+			By("Invoking virtctl vnc with --proxy-only")
+			proxyOnlyCommand := tests.NewVirtctlCommand("vnc", "--proxy-only", "--namespace", vmi.Namespace, vmi.Name)
+
+			r, w, _ := os.Pipe()
+
+			// Stdout should be set back to previous value after the test
+			oldStdout := os.Stdout
+			os.Stdout = w
```

Comment:
Instaed of this, can you please do `cmd.SetOut(w)` and remove the Stdout mangling of the process?

Reply:
Done

Reply:
I tried this initially with `io.Pipe()` which didn't seem to be working when passing writer to `cmd.SetOut`; but with `os.Pipe()` seems working fine currently